src/font/SharedGrid.zig

```diff
@@ -332,11 +332,12 @@ const GlyphKey = struct {
 
     const Context = struct {
         pub fn hash(_: Context, key: GlyphKey) u64 {
-            return @bitCast(Packed.from(key));
+            const packed_key = Packed.from(key);
+            return std.hash.CityHash64.hash(std.mem.asBytes(&packed_key));
```

Comment:
How is hashing the value improving throughput? This seems to be doing strictly more work.

Reply:
I would like to see the benchmark change of this specifically. It might improve uniformity, which will reduce clustering, but in that case we should probably use `std.hash.int`, rather than cityhash, since that's specifically what it's designed for.

Reply:
Thanks, thanks for the disclosure. I reviewed the benchmark and it isn't ideal but I think its perfectly adequate for the task at hand, and the changes are so minimal and human understandable I don't need it to be perfect.